### PR TITLE
docs: Add process "How to create delete a repo"

### LINKED ISSUE #92: [DOCS] Add process/guide - "How to create delete a repo"
### Description

Hey! Contributor,

This issue is for adding the guide/process for **How we can delete a repo**. The step needs to be well defined and include the Screenshot/Screen recording for better explanation. To get an overview of how the docs should look, check the available docs [`here`](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How-to/guide)

- Path to the dedicate Markdown for this issue - [`open-source-with-pradumna/pages/How-to/guide/delete-repo.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How-to/guide/delete-repo.md)

---

If you have any suggestions feel free to Open an [Issue](https://github.com/Pradumnasaraf/open-source-with-pradumna/issues)

**Also if you need any kind of help, feel free to ping!**

### PR DESCRIPTION
**Pull request by YashPimple**
closes #92

### CODE REVIEW

File: pages/How-to/guide/delete-the-repo.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### Deleting a repository
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/delete-the-repo.md:1]:
```suggestion
## Deleting a repository
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thanks for the PR. Need some minor changes

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inefficient

File: pages/How-to/guide/delete-the-repo.md (hunk 1 above)
Comment (kelvinparmar) [reply pages/How-to/guide/delete-the-repo.md]: Should I complete it?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution :tada:!!

**Pull request merged**